### PR TITLE
Make sure fluido used is 1.10.0, site.xml are consistent, and fix settings xml to latest xsd and correct tabs to spaces.

### PR DESCRIPTION
**Pull request by hazendaz**

### CODE REVIEW

File: oshi-core/src/site/site.xml [hunk 1]
```diff
@@ -25,28 +25,28 @@
 
 -->
 <project xmlns="http://maven.apache.org/DECORATION/1.8.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="${project.name}"
-	xsi:schemaLocation="http://maven.apache.org/DECORATION/1.8.0 http://maven.apache.org/xsd/decoration-1.8.0.xsd">
-	<skin>
-		<groupId>org.apache.maven.skins</groupId>
-		<artifactId>maven-fluido-skin</artifactId>
-		<version>1.7</version>
```
Comment (mprins) [inline oshi-core/src/site/site.xml:33]: if you define this version as a property in the pom eg. `<maven-fluido-skin.version>...</maven-fluido-skin.version>` and add it as a dependency of the `maven-site-plugin`  like

```xml
<plugin>
    <artifactId>maven-site-plugin</artifactId>
    <version>3.10.0</version>
    <dependencies>
        <dependency>
            <groupId>org.apache.maven.skins</groupId>
            <artifactId>maven-fluido-skin</artifactId>
            <version>${maven-fluido-skin.version}</version>
        </dependency>

   ...
```

you can use `${maven-fluido-skin.version}`  here and dependabot will keep it up-to-date.


example:
https://github.com/GeoDienstenCentrum/schemaspy-maven-plugin/blob/a6c06701115f98f2b7e4903a577908899587076a/pom.xml#L296-L311
and
https://github.com/GeoDienstenCentrum/schemaspy-maven-plugin/blob/a6c06701115f98f2b7e4903a577908899587076a/src/site/site.xml#L1-L12

File: src/site/site.xml [hunk 2]
```diff
@@ -1,13 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project
-    xmlns="http://maven.apache.org/DECORATION/1.8.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    name="${project.name}"
+<!--
+
+    MIT License
+
+    Copyright (c) 2010 - 2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```
Comment (mprins) [inline src/site/site.xml:6]:
```suggestion
    Copyright (c) 2010 - 2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```

File: oshi-dist/src/site/site.xml [hunk 3]
```diff
@@ -1,13 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project
-    xmlns="http://maven.apache.org/DECORATION/1.8.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    name="${project.name}"
+<!--
+
+    MIT License
+
+    Copyright (c) 2010 - 2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```
Comment (mprins) [inline oshi-dist/src/site/site.xml:6]:
```suggestion
    Copyright (c) 2010 - 2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```

File: oshi-demo/src/site/site.xml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    MIT License
+
+    Copyright (c) 2010 - 2021 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```
Comment (mprins) [inline oshi-demo/src/site/site.xml:6]:
```suggestion
    Copyright (c) 2010 - 2022 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
```

**Review by mprins — COMMENTED** [review]
if you are adding the copyright year it should be 2022

File: oshi-demo/src/site/site.xml (hunk 4 above)
Comment (dbwiddis) [reply oshi-demo/src/site/site.xml:6]: I actually decided some time back that xml files shouldn't be copyrighted.  This is in the [license-plugin excludes list](https://github.com/oshi/oshi/blob/b25a115eee3d79461a5c6c1bcfa52f4e1dd7d19c/pom.xml#L463).

Comment (hazendaz) [reply oshi-demo/src/site/site.xml:6]: no problem, I'll remove, squash and push back up.

File: oshi-core/src/site/site.xml (hunk 1 above)
Comment (hazendaz) [reply oshi-core/src/site/site.xml:33]: @mprins Nice!  Thank you that is much cleaner.  Will adjust to that.

File: oshi-demo/src/site/site.xml (hunk 4 above)
Comment (hazendaz) [reply oshi-demo/src/site/site.xml]: done, most were already copyrighted - there was just one file that was net new as missing.  Working on the dependabot ability now before this is complete.

File: oshi-core/src/site/site.xml (hunk 1 above)
Comment (hazendaz) [reply oshi-core/src/site/site.xml:33]: We were already tracking on site so versions plugin would note the update.  Dependabot was doing that bit as well as versions plugin is pretty useless in that regard now.  All the site pages have been updated.  I do believe technically speaking all sites inherit from each other so it is only really needed in the first but for simplicity given these are all the same, no reason to be different :)

File: oshi-demo/src/site/site.xml (hunk 4 above)
Comment (dbwiddis) [reply oshi-demo/src/site/site.xml]: Yeah, I haven't gone through to remove them yet.  The .properties file headers also eventually need to be axed.

**Review by dbwiddis — APPROVED** [review]
LGTM!

**Comment by dbwiddis** [comment]
Feel free to merge, LGTM.  If you want to fix the headers in the pom.xml either in this or another PR that's probably needed.  I used mvn tidy:pom thinking it was helpful but it's not.  Should be more https goodness.

**Comment by hazendaz** [comment]
Try the sortpom-maven-plugin.  Its gotten better and mostly default handles maven conventions set forth in 2006.

**Pull request merged**